gensim/models/ldamodel.py

```diff
@@ -182,6 +187,366 @@ def load(cls, fname, *args, **kwargs):
         return result
 # endclass LdaState
 
+# DD
+# wrapper to just return lda.state after performing lda
+from distributed import local_client
```

Comment:
don't forget to update `setup.py`

Reply:
Ok, I'll look what needs to be done there.